ispyb/__init__.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import absolute_import, division, print_function
 
+from contextlib import contextmanager
```

Comment:
import is not needed

Reply:
Thank you, and you're right, I forgot to remove it after trying a different approach. Have removed now, will merge.